logs/build.go

```diff
@@ -21,6 +28,8 @@ func (cmd *buildCmd) Run(ctx context.Context, client *api.Client) error {
 		}, client.Project)
 	}
 
+	since := time.Since(build.CreationTimestamp.Time)
+	cmd.Since = &since
```

Comment:
Same with the build get, this only works if we actually have a build name.

Reply:
added a check here aswell